x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/common/text/TextTemplateEngine.java

```diff
@@ -35,6 +35,12 @@ public String render(TextTemplate textTemplate, Map<String, Object> model) {
         String mediaType = compileParams(detectContentType(template));
         template = trimContentType(textTemplate);
 
+        int indexStartMustacheExpression = template.indexOf("{{");
+        int indexEndMustacheExpression = template.indexOf("}}");
+        if (indexStartMustacheExpression == -1 || indexStartMustacheExpression > indexEndMustacheExpression) {
```

Comment:
Why do we need to check for end brackets at all? Isn't `}}{{` and invalid mustache template?